src/scss/constants.scss

```diff
@@ -47,6 +47,8 @@ $room-info-bg: #005ee5;
 //   pages/Admin/Room/Modal/RoomModal.styles.ts (z-index: 101)
 //   components/molecules/UserProfilePicture/UserProfilePicture.styles.ts (z-index: 10)
 //   components/atoms/Legend/Legend.styles.ts (z-index: 5)
+$z-layer-navbar: 30;
+$z-layer-live-schedule: 20;
 $z-layer-sidebar: 15;
```

Comment:
[FYI] Starting to build up our layers. Ideally we would have them separated by at least increments of `10` for our 'main layers', and then we can use the digits in between for 'micro adjustments' as required.

[Future Cleanup] Eventually we can give these a better generic name based on the 'type of layer' it is, and probably reduce the number of 'specific names' we have in our `z()` helper. I expect we could also reduce the number of places where we even need to explicitly set a `z-layer` as we refactor/cleanup our code. In most situations I find that needing to use `z-index` is actually a bit of a hack.